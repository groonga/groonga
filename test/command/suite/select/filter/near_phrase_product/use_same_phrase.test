table_create Entries TABLE_NO_KEY
column_create Entries content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenNgram \
  --normalizer NormalizerNFKC150
column_create Terms entries_content COLUMN_INDEX|WITH_POSITION Entries content

load --table Entries
[
{"content": "a x a"},
{"content": "ab x ab"},
{"content": "abc x abc"},
{"content": "abc x x abc"}
]

select Entries \
  --filter 'content *NPP1 "(abc abc)"' \
  --output_columns '_id, content'
