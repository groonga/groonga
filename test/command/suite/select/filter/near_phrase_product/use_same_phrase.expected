table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR Text
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenNgram   --normalizer NormalizerNFKC150
[[0,0.0,0.0],true]
column_create Terms entries_content COLUMN_INDEX|WITH_POSITION Entries content
[[0,0.0,0.0],true]
load --table Entries
[
{"content": "a x a"},
{"content": "ab x ab"},
{"content": "abc x abc"},
{"content": "abc x x abc"}
]
[[0,0.0,0.0],4]
select Entries   --filter 'content *NPP1 "(abc abc)"'   --output_columns '_id, content'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        2
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "content",
          "Text"
        ]
      ],
      [
        3,
        "abc x abc"
      ],
      [
        4,
        "abc x x abc"
      ]
    ]
  ]
]
